### PR TITLE
Fix deprecation (implicit conversion) for PHP 8.1

### PR DESCRIPTION
**Pull request by petarmarj**
Migrate code to PHP 8.1 (fixing `PHP Deprecated:  Implicit conversion from float …`).

Unclear: Do Swiss country coordinates always have to be integers?

### CODE REVIEW

**Comment by petarmarj** [comment]
I fixed a few more deprecations. security-checker (see CONTRIBUTING-file) is no longer supported, psalm/codesniffer partly needed other config-files.

**Comment by WengerK** [comment]
Whoo thx @petarmarj I did not notice your contribution before today ..... very sorry for my very very late reply.

I will work on the project on the few next days to run tests on the CI, once done I will merge dev in your branch to ensure stability and merge if working <3

**Comment by WengerK** [comment]
> Unclear: Do Swiss country coordinates always have to be integers?

Having discussed with geo-professionals in Switzerland, we should indeed handle floating numbers. Still this library is not 100% accurate as we use an approximation to transform coordinates.

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,5 @@ composer.lock
 # PHPCS cache files
 .php_cs.cache
 .phpcs-cache
+composer.lock
```
Comment (WengerK) [inline .gitignore]:
```suggestion
```

already added on top of gitignore

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-0.0.1-alpha
+0.0.2-alpha
```
Comment (WengerK) [inline VERSION]:
```suggestion
0.0.1-alpha
```

keep it alpha 1 until we create the realse + tag (another PR)

Comment (WengerK) [inline VERSION]:
```suggestion
0.0.1-alpha
```

File: composer.json [hunk 3]
```diff
@@ -3,18 +3,18 @@
     "description": "Convert GPS (WGS84) to Swiss (LV03 and LV95) coordinates (and vice versa)",
     "license": "MIT",
     "require": {
-        "php": ">=7.4"
+        "php": ">=7.4 || >=8.1"
```
Comment (WengerK) [inline composer.json]:
```suggestion
        "php": ">=8.0"
```

**Pull request closed without merging**